apps/lottie-preview/src/locations/ConfigScreen.tsx

```diff
@@ -42,18 +47,21 @@ const ConfigScreen = () => {
 
       const fieldsByContentType = groupFieldsByContentType(jsonFieldsRef.current);
 
-      for (const [contentTypeId, allFields] of Object.entries(fieldsByContentType)) {
+      // Process all content types in parallel
+      const updatePromises = Object.entries(fieldsByContentType).map(async ([contentTypeId, allFields]) => {
```

Comment:
Similar performance improvement as aggregating the JSON fields for when we update the content types on config. This isn't as likely to be a problem since it'd be rare for someone to make a lot of updates in one go but why not make it better